doc/09-object-types.md

```diff
@@ -690,7 +690,8 @@ object Service "uptime" {
 
   check_command = "snmp"
 
-  vars.snmp_community = "public"
+  vars.snmp_comm
+  y = "public"
```

Comment:
What's this for?

Reply:
strange, haven't change this

Reply:
Are you sure?
https://github.com/Icinga/icinga2/pull/7953/commits/11300a6677e6fc36c769798f34f62bc150c50e06

Reply:
Maybe I accidentally added a New Line? I don't remember. How can I change this after this long time. Or can you change this?

Reply:
If I change it, the commit squashing I'm going to do is not guaranteed to work as expected.

You'll have to correct it [here](https://github.com/stevie-sy/icinga2/edit/patch-1/doc/09-object-types.md) and to click on _Commit changes_.